parent/pom.xml

```diff
@@ -1033,7 +1033,13 @@
 		<!-- ITests profile #1 of 3: won't run if -DskipITests=true -->
 		<profile>
 			<id>ITests</id>
+			<activation>
+				<property>
+					<name>!skipITests</name>
+				</property>
+			</activation>
 			<properties>
+				<skipITests>false</skipITests>
```

Comment:
Not sure you need to set skipITests=false when you've specifically said this profile will be activated unless skipITests=true. But I guess for clarity and completeness this is OK.